web/pgadmin/browser/server_groups/servers/databases/schemas/views/templates/views/ppas/15_plus/sql/create.sql

```diff
@@ -0,0 +1,32 @@
+{#============================Create new view=========================#}
+{% if display_comments %}
+-- View: {{ data.schema }}.{{ data.name }}
+
+-- DROP VIEW {{ conn|qtIdent(data.schema, data.name) }};
+
+{% endif %}
+{% if data.name and data.schema and data.definition %}
+CREATE{% if add_replace_clause %} OR REPLACE{% endif %} VIEW {{ conn|qtIdent(data.schema, data.name) }}
+{% if ((data.check_option and data.check_option.lower() != 'no') or data.security_barrier or data.security_invoker) %}
+WITH (
+{% if data.check_option and data.check_option.lower() != 'no' %}
+  check_option={{ data.check_option }}{% if data.check_option and data.check_option.lower() != 'no' and (data.security_barrier or data.security_invoker) %},
+{% endif %}{% endif %}
+{% if data.security_barrier %}
+  security_barrier={{ data.security_barrier|lower }}{% if data.security_barrier and data.security_invoker %},
```

Comment:
{% if data.security_barrier and data.security_invoker %} should be replaced by {% if data.security_invoker %}.
We are already inside "{% if data.security_barrier %}", so no need to check again.